README.md

```diff
@@ -18,9 +18,10 @@ Then add **hubot-youtube-tracker** to your `external-scripts.json`:
 ]
 ```
 
-## Sample Interaction
```

Comment:
Something to include in the future just to make it clear what it does. Can just copy, paste, and format it a little.
Also, we should include tests just for the listeners:

```coffee
expect(@robot.respond).to.have.been.calledWith(/ytt recent (.+)/i)
```